BoothDownloader/src/Program.cs

```diff
@@ -95,7 +97,7 @@ private static async Task<int> Main(string[] args)
             BoothConfig.Setup(configFile);
 
             #region First Boot
-            if (BoothConfig.Instance.Cookie == null)
+            if (string.IsNullOrWhiteSpace(BoothConfig.Instance.Cookie))
```

Comment:
I made this check null so string.Empty was essentially 'first boot complete, user didn't put in token'
This would start asking for a token every time.

Reply:
I changed it like this because if the token was null or failed token exchange, booth downloaded would set to the token to ""; and fail completely